bdanmaku.lua

```diff
@@ -79,6 +81,11 @@ function replace_sub()
 	end
 end
 
+function shutdown_handler()
+	os.execute('rm -f '..TMPDIR..mp.get_property('pid')..'.*')
```

Comment:
Doesn't work on Windows. Also, maybe no need to `-f`?

Reply:
[](https://github.com/UlyssesZh/bdanmaku/pull/1/commits/07d15007f351def44763f781e1e7f0c0d3f0d164)
Use lua os.remove function to fix the question.